extensions/sql-migration/src/wizard/assessmentDetailsPage/instanceSummary.ts

```diff
@@ -0,0 +1,248 @@
+export class InstanceSummary {
+	private _view!: azdata.ModelView;
+
+	// function to create instance summary components
+	public createInstanceSummaryContainer(view: azdata.ModelView): azdata.FlexContainer {
+		this._view = view;
+		const instanceSummaryContainer = view.modelBuilder.flexContainer().withLayout({
+			flexFlow: 'column'
+		}).withProps({
+			CSSStyles: {
+				'border-left': 'solid 1px'
+			}
+		}).component();
+
+		const heading = view.modelBuilder.text().withProps({
+			value: constants.DETAILS_TITLE,
+			CSSStyles: {
+				...styles.LABEL_CSS,
+				"padding": "5px",
+				"border-bottom": "solid 1px"
+			}
+		}).component();
+
+		const subHeading = view.modelBuilder.text().withProps({
+			value: constants.ASSESSMENT_SUMMARY_TITLE,
+			CSSStyles: {
+				...styles.LABEL_CSS,
+				"padding": "10px",
+				"border-bottom": "solid 1px"
+			}
+		}).component();
+
+		const description = view.modelBuilder.text().withProps({
+			value: constants.READINESS_SECTION_TITLE,
+			CSSStyles: {
+				...styles.SUBTITLE_LABEL_CSS,
+				'margin': '0px',
+				'padding-left': '10px'
+			}
+		}).component();
+
+		const assessedDatabasesLabel = view.modelBuilder.text().withProps({
+			value: constants.ASSESSED_DBS_LABEL + ":",
+			CSSStyles: {
+				...styles.LIGHT_LABEL_CSS,
+				'margin': '0px',
+				'padding-left': '10px'
+			}
+		}).component();
+
+		const findingsSummaryTitle = view.modelBuilder.text().withProps({
+			value: constants.INSTANCE_FINDING_SUMMARY,
+			CSSStyles: {
+				...styles.SUBTITLE_LABEL_CSS,
+				'margin': '0px',
+				'padding-left': '10px',
+				'margin-top': '30px'
+			}
+		}).component();
+
+		const totalFindingsLabel = view.modelBuilder.text().withProps({
+			value: constants.TOTAL_FINDINGS_LABEL + ":",
+			CSSStyles: {
+				...styles.LIGHT_LABEL_CSS,
+				'margin': '0px',
+				'padding-left': '10px'
+			}
+		}).component();
+
+		const findingsSummarySubtitle = view.modelBuilder.text().withProps({
+			value: constants.SEVERITY_FINDINGS_LABEL,
+			CSSStyles: {
+				...styles.SUBTITLE_LABEL_CSS,
+				'margin': '0px',
+				'margin-top': '10px',
+				'padding-left': '10px'
+			}
+		}).component();
+
+		const findings = [
+			{
+				label: constants.ISSUES_LABEL,
+				count: 0,
+				iconPath: IconPathHelper.error
+			},
+			{
+				label: constants.WARNINGS,
+				count: 0,
+				iconPath: IconPathHelper.warning
+			}];
+
+		instanceSummaryContainer.addItems([heading, subHeading, description, assessedDatabasesLabel, this.createGraphComponent(),
+			findingsSummaryTitle, totalFindingsLabel, findingsSummarySubtitle]);
+
+		instanceSummaryContainer.addItems(findings.map(l => this.createFindingsContainer(l)));
+		return instanceSummaryContainer;
+	}
+
+	private createGraphComponent(): azdata.FlexContainer {
+		const graph = this._view.modelBuilder.flexContainer().withProps({
+			CSSStyles: {
+				'display': 'flex',
+				'flex-direction': 'column',
+			}
+		}).component();
+
+		const labels = [
+			{
+				label: "Not ready",
```

Comment:
can this be replaced with a resource string

Reply:
done.